src/tokens/solana.tokenlist.json

```diff
@@ -25045,7 +25045,26 @@
         "discord": "https://discord.gg/gXymhAfceY",
         "twitter": "https://twitter.com/ottercoinn"
       }
-    }
+    },
```

Comment:
DON'T EDIT 
add your token details under },

Reply:
I'm sorry just saw this. Is it okay now?